src/main/java/seedu/address/model/task/Task.java

```diff
@@ -151,6 +151,16 @@ public int getNoOfPeople() {
         return this.people.size();
     }
 
+    /**
+     * Returns a copy-paste friendly string containing all the emails related to this task.
```

Comment:
Maybe can include an example on the format? Like returns a string in the format: e123456@u.nus.edu, e2345...

Reply:
donezo